Library/ViewModels/PledgeCTAContainerViewViewModel.swift

```diff
@@ -160,3 +157,14 @@ private func subtitle(project: Project, pledgeState: PledgeStateCTAType) -> Stri
   guard let rewardTitle = reward.title else { return "\(amount)" }
   return "\(amount) • \(rewardTitle)"
 }
+
+private func formattedPledge(amount: Double, project: Project) -> String {
```

Comment:
Hey @cdolm92, was there a reason for not using an `NSNumberFormatter` here?

Reply:
@justinswart Are we using it to format decimal values elsewhere?

Reply:
✅ Convo resolved on slack. The formatter will be added in separate pr.